### DISCUSSION TITLE
Manually building a running binary (fastschema)

### DISCUSSION THREAD

**Discussion by maen08** [fastschema/fastschema#142] [Ideas]
Hi there, 
Its easy to run the project with docker and with compiled binary. But can we improve the repo such that anyone can build the final binary on themselves? This will be nice for development and testing because even when running the project using docker still docker goes to fetch the compiled binary from github:

```
RUN wget https://github.com/fastschema/fastschema/releases/download/v${VERSION}/fastschema_${VERSION}_${BUILDX_ARCH}.zip \
    && unzip fastschema_${VERSION}_${BUILDX_ARCH}.zip \

```

How can I compile a binary "fastschema" on my own?

**Comment by ngocphuongnb** [comment]
@maen08 thanks for your question.
Fastschema can be built using `go build` command. Please following the guide here and tell me if you encountered any problem.
https://fastschema.com/docs/getting-started.html#method-3-build-from-source

**Reply by maen08** [reply]
Hi @ngocphuongnb  thank you for this. I guess i didnt see this. Thanks
